mozilla/foundation.mozilla.org
# fade effect for subcategory bar

# Description
Related PRs/issues: #9871 
Link to sample test page: http://localhost:8000/en/privacynotincluded/categories/smart-home/

This PR adds the fade effect on the end of the subcategories for medium sized screens and above. 
Figma designs can be found here: [Figma](https://www.figma.com/file/GwdoUiY4fzVyeLfXsAJGPq/*Privacy-Not-Included-Archive-2022?node-id=5609%253A102175)


# Screenshots


## L - XL:

**Beginning of list:**
![Screenshot 2023-02-01 at 17-41-29 Privacy Not Included A Buyer’s Guide for Connected Products](https://user-images.githubusercontent.com/18314510/216210355-22843070-4717-4b18-af5d-93e56be1cd97.png)


**End of list:**
![Screenshot 2023-02-01 at 17-41-53 Privacy Not Included A Buyer’s Guide for Connected Products](https://user-images.githubusercontent.com/18314510/216210364-615a4b21-3a71-4f8a-8971-91bcfa841be7.png)

## M

**Beginning of list:**
<img width="400" alt="206596143-62e265e7-45be-444e-acf8-c050dc5e0e7a" src="https://user-images.githubusercontent.com/18314510/216210453-fe37ced1-c8ac-4438-837d-bba09718739e.png">


**End of list:**
<img width="400" alt="206596143-62e265e7-45be-444e-acf8-c050dc5e0e7a" src="https://user-images.githubusercontent.com/18314510/216210466-7947256c-5698-4601-ba3c-504c55695fcb.png">

## S

**Beginning of list:**
<img width="400" alt="206596143-62e265e7-45be-444e-acf8-c050dc5e0e7a" src="https://user-images.githubusercontent.com/18314510/216207515-e037426a-db99-470e-a59c-45aae0e5d584.png">


**End of list:**
<img width="400" alt="206596143-62e265e7-45be-444e-acf8-c050dc5e0e7a" src="https://user-images.githubusercontent.com/18314510/216207518-9bc7bef9-90aa-4c31-9509-289d39c86360.png">


## XS

**Beginning of list:**

<img width="400" alt="206596143-62e265e7-45be-444e-acf8-c050dc5e0e7a" src="https://user-images.githubusercontent.com/18314510/216207388-8452b946-dfe3-4f0a-8898-ac5373b83c1f.png">

**End of list:**

<img width="400" alt="206596143-62e265e7-45be-444e-acf8-c050dc5e0e7a" src="https://user-images.githubusercontent.com/18314510/216207396-4e29162e-243d-4a93-a80b-177ceea76ead.png">



# Steps to test

1. Check this branch out
2. run `inv copy-prod-db
3. Visit http://localhost:8000/en/privacynotincluded/categories/smart-home/
4. If the subcategory nav below is looking as expected on all screen sizes, testing is complete!
